### DISCUSSION TITLE
What's the risk of a supply chain attack on Bisq?

### DISCUSSION THREAD

**Discussion by Conza88** [bisq-network/bisq#5188] [Q&A]
[https://twitter.com/jamesob/status/1360065911277957122](https://twitter.com/jamesob/status/1360065911277957122)

![image](https://user-images.githubusercontent.com/51952482/107728308-d8a62100-6d41-11eb-80b4-7923634e59e7.png)
![image](https://user-images.githubusercontent.com/51952482/107728317-dcd23e80-6d41-11eb-8e03-fb37941dec4c.png)

**Comment by ripcurlx** [comment]
This isn't an immediate issue for us as we are first not auto-updating dependencies and all dependencies that are resolved are compared to a hash to prevent any changes when they are resolved. It would be something to look for when we update libraries and update hashes for new versions.
